.devcontainer/db/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM postgres:alpine
+RUN apk upgrade --update-cache --available && \
+    apk add openssl && \
+    rm -rf /var/cache/apk/*
```

Comment:
`RUN apk update && apk add --no-cache openssl` should be enough... why should we upgrade?

Reply:
You're probably right.
I've copied that over from a container using edge.

Reply:
Fixed.